drivers/flash/spi_nor.c

```diff
@@ -702,6 +702,33 @@ static int spi_nor_read(const struct device *dev, off_t addr, void *dest,
 	return ret;
 }
 
+#if defined(CONFIG_FLASH_EX_OP_ENABLED)
+static int flash_spi_nor_ex_op(const struct device *dev, uint16_t code,
+				const uintptr_t in, void *out)
+{
+	int ret = 0;
```

Comment:
no need to initialize this, drop the init then the compiler should warn if there's any code path that uses this in an erroneous way